editor/editor_properties_vector.h

```diff
@@ -61,9 +61,14 @@ class EditorPropertyVectorN : public EditorProperty {
 	void _notification(int p_what);
 
 public:
+	enum VectorDisplayStyle {
+		VERTICAL,
+		HORIZONTAL,
+		COMPACT
+	};
```

Comment:
Use Godot-style enum naming. No need to specify Vector as we're already within a Vector-themed class, and prefix each value with the uppercased enum name (for scoping).

```suggestion
	enum DisplayStyle {
		DISPLAY_STYLE_VERTICAL,
		DISPLAY_STYLE_HORIZONTAL,
		DISPLAY_STYLE_COMPACT,
	};
```